### PR TITLE
Default configuration legacy

### PR DESCRIPTION
**Pull request by yuhao900914**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

[JavaScritp legacy SDK](https://pr-666.d19s7xzcva2mw3.amplifyapp.com/data/sdks/javascript/)
[Node legacy SDK](https://pr-666.d19s7xzcva2mw3.amplifyapp.com/data/sdks/node/)
[React Native legacy SDK](https://pr-666.d19s7xzcva2mw3.amplifyapp.com/data/sdks/react-native/)

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sdks/flutter/index.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ class YourClass {
 Amplitude Flutter SDK runs on the top of the [Amplitude Android Mantaince SDK](../android/), [Amplitude iOS Mantaince SDK](../ios/) and [Amplitude JavaScript Mantaince SDK](../javascript/). The following are the Dart settable config options.
 For other default configurations:
 
-- on Android side, check the [Android Configuration](../android-kotlin/#configuration)
+- on Android side, check the [Android Configuration](../android/#configuration)
```
Comment (Mercy811) [inline docs/data/sdks/flutter/index.md:67]: Good catch!

File: docs/data/sdks/javascript/index.md [hunk 2]
```diff
@@ -5,17 +5,16 @@ icon: simple/javascript
 ---
 
 <!-- vale off-->
-[![npm version](https://badge.fury.io/js/amplitude-js.svg)](https://badge.fury.io/js/amplitude-js)
+[![npm version](https://badge.fury.io/js/amplitude-js.svg)](https://www.npmjs.com/package/amplitude-js)
```
Comment (Mercy811) [inline docs/data/sdks/javascript/index.md:8]: where did you get the new one? I got the link the same as our old one. 
image

File: docs/data/sdks/node/index.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ description: The Amplitude Node.js SDK installation and quick start guide.
 icon: simple/nodedotjs
 ---
 
-![npm version](https://badge.fury.io/js/%40amplitude%2Fnode.svg)
+[![npm](https://img.shields.io/npm/v/@amplitude/node)](https://www.npmjs.com/package/@amplitude/node)
```
Comment (Mercy811) [inline docs/data/sdks/node/index.md:7]: This is what I got
```
[![npm version](https://badge.fury.io/js/@amplitude%2Fnode.svg)](https://badge.fury.io/js/@amplitude%2Fnode)
```

File: docs/data/sdks/react-native/index.md [hunk 4]
```diff
@@ -6,17 +6,15 @@ icon: simple/react
 
 <!-- vale off-->
 
-![npm version](https://badge.fury.io/js/%40amplitude%2Freact-native.svg)
+[![npm](https://img.shields.io/npm/v/@amplitude/react-native)](https://www.npmjs.com/package/@amplitude/react-native)
```
Comment (Mercy811) [inline docs/data/sdks/react-native/index.md:9]: The same as above
```
[![npm version](https://badge.fury.io/js/@amplitude%2Freact-native.svg)](https://badge.fury.io/js/@amplitude%2Freact-native)
```

**Review by Mercy811 — APPROVED** [review]

File: docs/data/sdks/node/index.md (hunk 3 above)
Comment (yuhao900914) [reply docs/data/sdks/node/index.md:7]: 1. the badge.fury.io has an issue in the doc. This comes from https://shields.io/. Please check this thread for the discussion. https://amplitude.slack.com/archives/C02T742UUMC/p1678302120754409
2. Justin pointed out, we should redirect the customer to the library, for example, redirect to npm for ts, and redirect to maven central to android. I noticed it works right in the readme.md doc in each GitHub. For example, https://github.com/amplitude/Amplitude-Kotlin/edit/main/README.md. I noticed it's not able to redirect in the staging environments. So I want to test it out on production.

File: docs/data/sdks/javascript/index.md (hunk 2 above)
Comment (yuhao900914) [reply docs/data/sdks/javascript/index.md:8]: This should be updated to use `img.shields.io`.

Comment (yuhao900914) [reply docs/data/sdks/javascript/index.md]: Updated.

**Review by justin-fiedler — APPROVED** [review]
Thanks @yuhao900914. This is super helpful!

**Review by qingzhuozhen — APPROVED** [review]

**Pull request merged**